lib/puppet/vendor/semantic_puppet/lib/semantic_puppet.rb

```diff
@@ -1,15 +1,7 @@
-begin
-  require 'gettext-setup'
-rescue LoadError
-  def _(msg)
-    msg
-  end
-end
+require 'gettext-setup'
 
 module SemanticPuppet
-  if defined?(GettextSetup)
-    GettextSetup.initialize(File.absolute_path('semantic_puppet/locales', File.dirname(__FILE__)))
-  end
```

Comment:
I think we might need to keep this feature gate? It was introduced separately in https://github.com/puppetlabs/puppet/commit/0016250c